index.js

```diff
@@ -156,6 +156,15 @@ var extensions = {
         });
       },
     },
+    {
+      module: '@swc/register',
+      register: function(hook) {
+        hook({
+          extensions: '.ts',
+          ignore: [ignoreNonBabelAndNodeModules],
```

Comment:
Assuming those changes are released, we'll want to evaluate supporting `.swc.js` and changing this ignore to one specific for swc.